### ISSUE TITLE
Is it possible to make an arm64v8 version in dockerhub?

### ISSUE THREAD

**Issue by ProblemFactory** [screepers/screeps-launcher#23]
Hi, I am trying to deploy this in my raspberry pi with docker but the docker image only supports amd64. Can you make a version for arm64v8? Thank you.

**Comment by ags131** [comment]
I'll have to look into doing an arm build via circleci, haven't done that recently. For now, an option is to build your own. `docker build -t screeps-launcher .` in the folder on an arm machine should work.

**Comment by Elbarae1921** [comment]
Make sure to change this line in the Dockerfile tho 
```diff
RUN CGO_ENABLED=0 \
    GOOS=linux \
-   GOARCH=amd64 \
+   GOARCH=arm64 \
    go build -o screeps-launcher ./cmd/screeps-launcher
```
Otherwise you'll just be building an amd64 image.

**Comment by philipp-horstenkamp** [comment]
The newest version contains something similar on and build arg in Dockerfile / compose. This may be enough?

**Comment by Elbarae1921** [comment]
yes as long as you specify `ARCH: arm64` in the compose file. or change the default value in the Dockerfile. that should work

**Comment by philipp-horstenkamp** [comment]
Should be working now
